tsconfig.json

```diff
@@ -127,6 +133,9 @@
     {
       "path": "./examples/cactus-example-discounted-asset-trade/tsconfig.json"
     },
+    {
+      "path": "./examples/cactus-example-fujitsu-microfinance/tsconfig.json"
```

Comment:
@ervvkb This file does not exist on the main branch, nor in your PR diff. Was this added by mistake?

Reply:
I removed the path to microfinance, the second path is correct, it just wasn't merged before uploading the gui

Reply:
> I removed the path to microfinance, the second path is correct, it just wasn't merged before uploading the gui

@ervvkb Got it, thank you! Now in order to make it a little easier to review the large diff, please re-organize the commits the same way I explained it here in this other PR as well: https://github.com/hyperledger/cacti/pull/2331#pullrequestreview-1360124392